MapboxNavigation/JunctionView.swift

```diff
@@ -32,14 +33,18 @@ public class JunctionView: UIImageView {
         
         if quaternaryInstruction == nil {
             hide(delay: 10, animated: true)
+            distanceAlongStep = nil
         }
         
         guard let guidanceView = quaternaryInstruction?.components.first else { return }
-        
+        if let visualInstructionDistance = visualInstruction?.distanceAlongStep {
+            distanceAlongStep = visualInstructionDistance
+        } else {
+            distanceAlongStep = nil
+        }
```

Comment:
Nit: `distanceAlongStep` is already optional, so we don’t need to unwrap it:

```suggestion
        distanceAlongStep = visualInstruction?.distanceAlongStep
```